backend/server/db/helpers/session_tokens.py

```diff
@@ -0,0 +1,75 @@
+from time import time
+from typing import Optional
+from uuid import UUID
+
+from server.db.redis.conn import sdb
+from redis.commands.search.query import Query
+from .models import SessionID, SessionToken, SessionTokenInfoModel
+
+# FT.CREATE idx:uid ON HASH PREFIX 1 "stoken:" NOOFFSETS NOHL NOFIELDS NOFREQS STOPWORDS 0 SCHEMA uid TAG CASESENSITIVE
+# FT.SEARCH idx:uid "@uid:{z5362383}" NOCONTENT VERBATIM
+# FT.SEARCH idx:sid "@sid:{bf362dd3\\-046a\\-49e4\\-8d63\\-fd379f06a40f}" NOCONTENT VERBATIM
+# HMGET stoken:Nj-C2n8JYS4imxlVn7WhSX5Pa0uRZ1awa1w_YA-vnN4jMXXqqKVb7HW9rNhfGako sid uid
```

Comment:
Are these meant to still be here? What do they do?

Reply:
oh this is just for self reference later, since its kinda hard to figure out these queries

Reply:
could probs remove

Reply:
they already are (the functions in the file are implementations of them), was moreso keeping them around coz i use them in debugging... I will remove after merge when we revisit the indexing (i want to get rid of redis search its not appropriate)